lizzy/apps/senza.py

```diff
@@ -176,3 +176,24 @@ def render_definition(self, senza_yaml: str, stack_version: str,
                 self.logger.error('Failed to render CloudFormation defition.',
                                   extra={'command.output': exception.output})
                 raise SenzaRenderError(exception.error, exception.output)
+
+    def scale(self, stack_name: str, stack_version: str, new_scale: int):
+        """
+        Rescales the application.
+
+        :param stack_name: Name of the application stack
+        :param stack_version: Name of the application version that will be
+                              changed
+        :param scale: New scale
+        :raises SenzaScaleError: when a ExecutionError is thrown to allow
+                                   more specific error handing.
+        """
+        try:
+            arguments = []
+            if new_scale is not None:
+                arguments.append(str(new_scale))
+            arguments.append('--force')
+
+            return self._execute('scale', stack_name, stack_version, *arguments)
```

Comment:
If `new_scale` is `None`, we might as well not execute anything.

Reply:
Has been fixed in line 195 - please check